chapters/VK_EXT_acquire_drm_display/acquire_drm_display.txt

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018-2021 The Khronos Group, Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[open,refpage='vkAcquireDrmDisplayEXT',desc='Acquire access to a VkDisplayKHR using DRM',type='protos']
+--
+
+To acquire permission to directly a display in Vulkan from the Direct Rendering
+Manager (DRM) interface, call:
+
+include::{generated}/api/protos/vkAcquireDrmDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device the display is on.
+  * pname:drmFd Connection to the DRM node file descriptor.
```

Comment:
This still reads "connection". It should just read "DRM primary file descriptor".

Reply:
Ideally also mention that the DRM FD must have DRM master permissions.